usr/share/rear/restore/NFS_SERVER/default/300_start_nfs_server.sh

```diff
@@ -0,0 +1,35 @@
+# 300_start_nfs_server.sh
+
+# same options works for mountd
+local nfsd_opts="--no-udp --no-nfs-version 3"
```

Comment:
We use Bash arrays for multi value strings to be on the safe side

```suggestion
local nfsd_opts=( --no-udp --no-nfs-version 3 )
```